Unix/xmlserializer/xmldeserializer_ids.h

```diff
@@ -27,17 +27,17 @@
 #define ID_MI_DES_XML_INST_TOO_MANY_CLASSES             PAL_T("Failed to deserialize CIM-XML document. The instance is too complex to deserialize because it uses too many classes.")
 #define ID_MI_DES_XML_INST_CANNOT_FIND_CLASS            PAL_T("Failed to deserialize CIM-XML document. Could not find the class definition %T while deserializing instance.")
 #define ID_MI_DES_XML_CLASS_DERIVATION_LIST_WRONG       PAL_T("Failed to deserialize CIM-XML document. A class definition has a SUPERCLASS attribute and a DERIVATION attribute. The SUPERCLASS attribute value should be the first item in the DERIVATION attribute list." )
-#define ID_MI_DES_XML_CLASS_SUPERCLASS_PARENT_MISSMATCH PAL_T("Failed to deserialize CIM-XML document. While decoding a class definition there was a mismatch between the SUPERCLASS attribute and the passed in parent class definition." )
+#define ID_MI_DES_XML_CLASS_SUPERCLASS_PARENT_MISMATCH PAL_T("Failed to deserialize CIM-XML document. While decoding a class definition there was a mismatch between the SUPERCLASS attribute and the passed in parent class definition." )
```

Comment:
i try to avoid harming whitespace indentation, but I lost this -- I saw it in one pass, but the page reloaded and I couldn't find it